source/renderer/zettlr-attachments.js

```diff
@@ -188,6 +189,8 @@ class ZettlrAttachments {
               console.error('Could not open attachment:' + potentialError)
             }
           })
+      } else {
+        shell.openExternal(elem.attr('href'))
```

Comment:
Please add a short comment explaining that this code is invoked if the user clicks on a link coming directly from bib data (e.g. url or doi).

Reply:
Sounds good, done!